internal/pkg/cli/init.go

```diff
@@ -343,6 +345,25 @@ func newInitOpts(vars initVars) (*initOpts, error) {
 			return nil
 		},
 		useExistingWorkspaceForCMDs: useExistingWorkspaceClient,
+		tryUseExistingWorkspaceForValidation: func(o *initOpts) {
```

Comment:
what do you think of we try `workspace.Use(fs)` inside of  `newInitOpts()` , and initialize all commands with the workspace if it succeeds?

Reply:
I feel like this might be pointless though, just because there's a chance the workspace is still created with `app init`, in which case we still need this.

Reply:
Discussed offline too!